clickhouse-jdbc/src/main/java/com/clickhouse/jdbc/PreparedStatementImpl.java

```diff
@@ -1,306 +1,397 @@
 package com.clickhouse.jdbc;
 
+import com.clickhouse.logging.Logger;
```

Comment:
I would like to go away from proprietary wrappers for very standard things. 
There is SLF4J that works perfectly. 
I think it is important to change it now because later it will require to cut off too many weed.